### ISSUE TITLE
TopBar Does not go away.

### ISSUE THREAD

**Issue by mnaamani** [Joystream/apps#150]
After merging https://github.com/Joystream/apps/pull/140

The call to `MyAccountContainer.hasAddress` is returning undefined, used in TopBar from https://github.com/Joystream/apps/blob/705552ce20950a4638b578d3a205728815cbcbfd/packages/apps/src/TopBar.tsx#L36

This is because calls in
https://github.com/Joystream/apps/blob/705552ce20950a4638b578d3a205728815cbcbfd/packages/joy-utils/src/index.ts#L51

to `getAccount` and `getAddress` on keyring are throwing

```
Error: Keyring should be initialised via 'loadAll' before use
    at Keyring.get keyring [as keyring] (Base.js?692c:51)
    at Keyring.decodeAddress (Base.js?692c:55)
    at Keyring.getAddress (Keyring.js?1512:128)
    at Keyring.getAccount (Keyring.js?1512:115)
    at findNameByAddress (index.ts?592f:80)
    at isKnownAddress (index.ts?592f:95)
    at MyAccountContainer.hasAddress (MyAccount.tsx?174e:56)
    at eval (TopBar.tsx?7c68:70)
    at eval (unstated.es.js?0b9b:150)
    at Consumer.render (index.js?b30d:133)
```

The keyring is initialized in https://github.com/Joystream/apps/blob/705552ce20950a4638b578d3a205728815cbcbfd/packages/ui-api/src/Api.tsx#L112

So the error above suggests we are not importing the same keyring in `joy-utils/src/index.ts`

**Comment by mnaamani** [comment]
From console logging i see
1. the import of keyring is happening first in joy-utils/src/index.ts, 
2. keyring.loadAll() is called in ui-api/src/Api.tsx
3. and finally first call to findNameByAddress() 

so at that point the error "Keyring should be initialised via 'loadAll' before use" is strange and could only happen if two separate instances of kerying are being loaded.

**Comment by mnaamani** [comment]
Fixed in #203
